src/xdocs/index.xml.vm

```diff
@@ -96,6 +96,15 @@
         For the exact day, please follow our
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
+      <p>
+        Checkstyle is following Emotional Semantic Versioning:
+        <b>First</b>.<b>Second</b>.<b>Third</b>
+        <br/>
+        <b>First</b> digit means that some very noticable breaking compatibility happened.<br/>
+        <b>Second</b> digit means that there are some breaking compatibility happened
+        or new features/modules introduced.<br/>
+        <b>Third</b> digit means that just defects are fixed.
+      </p>
       <p>
         Checkstyle is not using Semantic Versioning
```

Comment:
I think we should keep it, to avoid never ending requests to switch to semiver

Reply:
Romantic is not well defined, it is term of "not-semiver".
Might be better to remove Romantic at all to avoid confusion.

@nrmancuso , do you agree ? Or should we continue with defining Romantic ?

Reply:
We need to define some term for how we handle versioning. "Romantic" and "Sentimental" versioning terms already exist and aptly describe what we do. We can make up some other name, but I do not see any benefit in this.  Let's add a link from https://github.com/checkstyle/checkstyle/pull/12581#discussion_r1060270812 to above "Romantic Versioning" and drop "Semantic" from it.

`Checkstyle is following Romantic Semantic Versioning` -> `Checkstyle is following [Romantic Versioning](https://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/)`

Reply:
it should stay.
Changed wording.